corehq/apps/domain/forms.py

```diff
@@ -1157,7 +1157,7 @@ def process_submission(self, domain=None):
             }
             html_content = render_to_string("domain/email/pro_bono_application.html", params)
             text_content = render_to_string("domain/email/pro_bono_application.txt", params)
-            recipient = settings.BILLING_EMAIL
+            recipient = settings.SUPPORT_EMAIL
             subject = "[Pro-Bono Application]"
             if domain is not None:
                 subject = "%s %s" % (subject, domain)
```

Comment:
This is inside `ProBonoForm. process_submission` if that's not clear from the diff